src/gt4py/next/ffront/foast_to_gtir.py

```diff
@@ -406,7 +406,12 @@ def create_if(
 
         return im.let(cond_symref_name, cond_)(result)
 
-    _visit_concat_where = _visit_where  # TODO(havogt): upgrade concat_where
+    def _visit_concat_where(self, node: foast.Call, **kwargs: Any) -> itir.FunCall:
+        domain, true_branch, false_branch = self.visit(node.args, **kwargs)
+        # TODO: use this case again. breaks domain inference in fused_velocity_advection_stencil_1_to_7
+        #  because some tuple elements are never accessed and the collapse tuple
+        #  does not propagate across concat where
```

Comment:
```suggestion
```
seems this is a leftover of https://github.com/GridTools/gt4py/commit/4764c2be09cea5f9c126c301d8ae71b6adfb3c07

Reply:
removed